### ISSUE TITLE
People: Untagged faces section

### ISSUE THREAD

**Issue by graciousgrey** [photoprism/photoprism#1556]
**As user, I want a view with all faces that are not tagged and do not belong to a cluster, so that I do not need to search those.**

Acceptance Criteria:
* Add a view (like the one for new face clusters) that shows faces that do not belong to a cluster and are not tagged yet

**Comment by alexislefebvre** [comment]
It looks like it's already implemented? https://demo.photoprism.org/people/new

**Comment by lastzero** [comment]
It only shows recognized face clusters, not ALL faces independent of being part of a cluster or even having a valid embedding.

**Comment by danielo515** [comment]
> It looks like it's already implemented? https://demo.photoprism.org/people/new

That section doesn't show anything for me. It says 2k faces found, and then the page it's empty

**Comment by lastzero** [comment]
In that case 2k faces may cause performance issues and it's just loading slow. How many files do you have? Do you collect photos of people professionally or how do you have so many face clusters?

**Comment by lastzero** [comment]
You may tweak facial recognition parameters in [config options](https://docs.photoprism.org/getting-started/config-options/), delete existing clusters and reindex. For example, you could increase `PHOTOPRISM_FACE_SCORE` and `PHOTOPRISM_FACE_CLUSTER_CORE`.

**Comment by danielo515** [comment]
> In that case 2k faces may cause performance issues and it's just loading slow. How many files do you have? Do you collect photos of people professionally or how do you have so many face clusters?

Really? I have a project that uses the same pictures and I can find, save and load all the faces,so I don't think it is a performance issue. What I think it may be happening is that my project (which is designed to run along with photoprism) may be clashing with the tables photoprism uses. How are the tables sotoring the faces named? I think because I have the logs full of errors like this `marker: face 1180 not found` , so my guess is that we are using the same table names.

**Comment by lastzero** [comment]
markers, faces, subjects are the table names

**Comment by danielo515** [comment]
Yep, it was a clash between photoprism and my project table names. I am surprised how similar our tables where (apart from the fact that mine had 128 extra columns)

**Comment by joachimtingvold** [comment]
Photos on `people/new` -> "Show all new faces" also disappear if at least one face has been tagged, and you re-do the search. I.e. if a photo has 4 faces, and you only assign one of them (with 3 remaining), the photo is no longer shown on the "Show all new faces"-query. I guess this request would cover that scenario.

**Comment by lastzero** [comment]
Works as designed as you may never know everyone.

**Comment by joachimtingvold** [comment]
Sure. It was just one more example that is currently not covered, and would be solved whenever the topic of this issue is resolved (I assume, as those faces would then be not tagged).

**Comment by graciousgrey** [comment]
@joachimtingvold Yes, the use case you mention will be covered once this issue is  solved :)

**Comment by ark-** [comment]
Is there anyway for you to tag a few of the unclustered photos and recalculate the clustering based on this info? Or would that require re-training the model?

**Comment by graciousgrey** [comment]
@ark- Each time the system is getting new information such as a user naming or renaming faces, the background worker adapts the face clusters accordingly
